pkg/ocm/clusters.go

```diff
@@ -197,7 +197,18 @@ func (c *Client) GetClusters(creator *aws.Creator, count int) (clusters []*cmv1.
 }
 
 func (c *Client) GetAllClusters(creator *aws.Creator) (clusters []*cmv1.Cluster, err error) {
-	return c.GetClusters(creator, 0)
+	query := getClusterFilter(creator)
+	request := c.ocm.ClustersMgmt().V1().Clusters().List().Search(query)
+	response, err := request.Send()
+
+	if err != nil {
+		return clusters, err
+	}
+	response.Items().Each(func(cluster *cmv1.Cluster) bool {
```

Comment:
Do we need to loop through the response, can we not return the slice from the response 

```
        query := getClusterFilter(creator)
	request := c.ocm.ClustersMgmt().V1().Clusters().List().Search(query)
	response, err := request.Send()
	if err != nil {
		return clusters, err
	}
	return response.Items().Slice(), nil
```